integration_test/third_party_apps_data/agent/ops-agent/linux/supported_applications.txt

```diff
@@ -1,10 +1,11 @@
 apache
 cassandra
+elasticsearch
```

Comment:
There are still some pending PRs for `elasticsearch`. Would this trigger a test failure by including `elasticsearch` early?

Reply:
@qingling128 i didn't actually add elasticsearch. i just moved it to an alphabetical spot

Reply:
Oops. Missed the last line. I see. Resolving this.